### PR TITLE
Repo cleanup

### PR DESCRIPTION
**Pull request by BearDooks**
This pull request changes a few main items

* Change Chuck's information to use his name and link to Github profile
* Move the original Chat Demo into it's own folder. This will make it easier to create and store demos in the future
* Remove the .gitmodules file as it is no longer used
* Edit the cloning commands to reflect repo changes
* Edit the LICENSE to reflect changes

### CODE REVIEW

**Review by WolfgangSenff — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -10,15 +10,21 @@ The following individuals and many more have contributed significantly to this p
 
 - Kyle Szklenski (creator) ([buy me a coffee](https://ko-fi.com/kyleszklenski))
 - Nicolò Santilio (creator)
```
Comment (fenix-hub) [inline README.md:12]: Replace `Nicolò Santilio`
With: 
```
[Nicolò Santilio](https://github.com/fenix-hub)
```

File: README.md [hunk 2]
```diff
@@ -10,15 +10,21 @@ The following individuals and many more have contributed significantly to this p
...
-- BearDooks
+- [Chuck Lindblom](https://github.com/BearDooks)
 - [Luke Hollenback](https://github.com/lukehollenback) ([buy me a coffee](https://ko-fi.com/lukehollenback))
```
Comment (fenix-hub) [inline README.md:14]: Eventually, the list could be replaced with:
```
<table>
  <tr>
    <th><img src='https://avatars0.githubusercontent.com/u/148612?s=460&u=30bd88ceee288169e9db370dccf5f26b73fa3fb6&v=4' alt='@WolfgangSenff' width="64"/><br/><a href="https://github.com/WolfgangSenff">Kyle Szklenski</a></th>
    <th><img src='https://avatars1.githubusercontent.com/u/45271396?s=460&u=914de9b89486317facffbf6ed6b4cbd2c241efc3&v=4' alt='@fenix-hub' width="64"/><br/><a href="https://github.com/fenix-hub">Nicolò Santilio</a></th> 
    <th><img src='https://avatars3.githubusercontent.com/u/20961993?s=460&u=48a1f0144df3162db23136868356936d8fbf5527&v=4' alt='@BearDooks' width="64"/><br/><a href="https://github.com/BearDooks">Chuck</a></th> 
    <th><img src='https://avatars1.githubusercontent.com/u/8144622?s=460&u=dc7b9d103bde9e13b2a00e747be87704d509ab46&v=4' alt='@lukehollenback' width="64"/><br/><a href="https://github.com/lukehollenback">Luke Hollenback</a></th> 
  </tr>
</table>
```

Result:
<table>
  <tr>
    <th>@WolfgangSenff<br/><a href="https://github.com/WolfgangSenff">Kyle Szklenski</a></th>
    <th>@fenix-hub<br/><a href="https://github.com/fenix-hub">Nicolò Santilio</a></th> 
    <th>@BearDooks<br/><a href="https://github.com/BearDooks">Chuck</a></th> 
    <th>@lukehollenback<br/><a href="https://github.com/lukehollenback">Luke Hollenback</a></th> 
  </tr>
</table>

**Review by fenix-hub — CHANGES_REQUESTED** [review]

**Review by fenix-hub — APPROVED** [review]

**Review by fenix-hub — CHANGES_REQUESTED** [review]
A little style change

**Review by lukehollenback — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (BearDooks) [reply README.md]: Change has been commited

**Review by fenix-hub — APPROVED** [review]

**Pull request merged**